lambdas/service/app.py

```diff
@@ -480,7 +480,7 @@ def get_manifest():
 
 def get_format(params):
     format_ = params.get('format', 'tsv')
-    if format_ in ('tsv', 'bdbag'):
+    if format_ in ('tsv', 'bdbag', 'metadata'):
```

Comment:
```suggestion
    if format_ in ('tsv', 'bdbag', 'full'):
```

This is in anticipation of https://github.com/DataBiosphere/azul/issues/1086 which I just created.

When naming methods in the service, prefer to use the term "full" instead of "metadata". Don't change the naming on the indexer side. From now on, think of the functionality as the "full manifest".